go/ql/test/library-tests/semmle/go/dataflow/VarArgsWithFunctionModels/Flows.ql

```diff
@@ -86,6 +82,8 @@ class TaintFlowTest extends InlineExpectationsTest {
     )
   }
 }
+
+import MakeTest<MergeTests<DataFlowTest, TaintFlowTest>>
 // from TaintConfiguration cfg, DataFlow::PartialPathNode source, DataFlow::PartialPathNode sink
 // where
 //   cfg.hasPartialFlow(source, sink, _)
```

Comment:
We can delete these lines now. But I wouldn't bother doing it unless you're also making another change - no point dismissing all the approvals.

Reply:
Yeah, I saw them, but wasn't sure if I should delete them.

> But I wouldn't bother doing it unless you're also making another change - no point dismissing all the approvals.

Agreed.